digicampipe/fit_cones_position.py

```diff
@@ -0,0 +1,44 @@
+from digicampipe.image.cones_image import ConesImage
+
+
+def get_cone_position_simu(output_dir=None):
+    cones_img = ConesImage('test', output_dir=output_dir)
+    cones_img.plot_cones(output_dir=output_dir)
+    cones_img.plot_fft_cones(output_dir=output_dir)
+    cones_img.get_cones_separation_reciprocal(output_dir=output_dir)
+    cones_img.plot_fft_cones(output_dir=output_dir, radius_mask=2.1)
+    cones_img.get_cone(output_dir=output_dir, radius_mask=2.1)
+    cones_img.scan_cone_position(output_dir=output_dir,
+                                 radius_mask=2.1,
+                                 center_scan=cones_img.center_fitted,
+                                 rotations=(60, 120, 180, 240, 300))
+    cones_img.plot_cones(output_dir=output_dir, radius_mask=2.1)
+    cones_img.plot_cones_presence(output_dir=output_dir)
+    cones_img.fit_camera_geometry()
+    cones_img.refine_camera_geometry()
+    cones_img.plot_camera_geometry(output_dir=output_dir)
+    return cones_img.pixels_pos_predict, cones_img.pixels_pos_true
+
+
+def get_cones_position(filename, output_dir=None):
+    cones_img = ConesImage(filename, './tests/resources/cones_1509411741-cone.fits')
```

Comment:
I think for this path here: `./tests/resources/....`  `pkg_resources` should be used.